client/components/OperationList.js

```diff
@@ -71,8 +71,12 @@ class CategorySelectComponent extends React.Component {
     }
 }
 
-function ComputeAttachmentLink(opId) {
-    return `operations/${opId}/file`;
+function ComputeAttachmentLink(op) {
+    let file = 'file';
+    if (op.binary.fileName !== null) {
```

Comment:
I don't see why :disappointed: , is it a cleaner way to detect there is no "fileName" parameter form binary?